ports/geckolib/gecko_bindings/bindings.rs

```diff
@@ -251,7 +253,7 @@ extern "C" {
                                           classList: *mut *mut *mut nsIAtom)
      -> u32;
     pub fn Gecko_GetServoDeclarationBlock(element: *mut RawGeckoElement)
-     -> *mut ServoDeclarationBlock;
+     -> ServoDeclarationBlockBorrowed;
```

Comment:
OK, then how should I fix this... I cannot figure out an easy way to make bindgen do something sensible.